js/custom/script.js

```diff
@@ -126,4 +126,17 @@
     });
   }
 
-})(jQuery, Drupal);
+  // Trigger page refresh when the shipping method changes
+  if($('.form-radio').length && window.location.pathname.indexOf('/checkout/') > -1) {
+    $('input[class="form-radio"]').change(function(e) {
+      setTimeout(window.location.reload(), 500);
+    });
+  }
+
+  if($('#edit-shipping-information-recalculate-shipping').length) {
+      $('body').on('blur', 'input.postal-code', function () {
+          console.log('blur!');
+          $('input[id^=edit-shipping-information-recalculate-shipping]').mousedown();
+      });
+  }
```

Comment:
I'm *sure* this hack isn't the solution we want (it may not work the same on sites that have been customized and it doesn't help other users of commerce_shipping who could use the same feature), but I'm not really sure what the proper solution is. See https://www.drupal.org/project/commerce_shipping/issues/2898118. Bojan described this as "not easy" but maybe changes to Commerce in the last few months have made things better.

It would be a good idea to get some direction from Bojan in the commerce channel of Drupal Slack (https://www.drupal.org/slack) about how to implement what we want (fixing it upstream in commerce or commerce_shipping rather than just in Drupal Orange).